build-rpi-raspbian/first-boot.sh

```diff
@@ -41,7 +41,10 @@ echo "watch cat /boot/first-boot.log" > "/home/$username/.bash_profile"
 if [ -z "${wifi_ssid}" ]; then
   echo "$(timestamp) [openHABian] Setting up Ethernet connection... OK"
 elif grep -q "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf; then
-  echo "$(timestamp) [openHABian] Setting up Wi-Fi connection... OK"
+  echo "$(timestamp) [openHABian] Setting up Wi-Fi connection... "
+  rfkill unblock wifi
```

Comment:
I worry which effect this line has on a normal RPi3 or RPi0W...

Reply:
since there are no problems with RPi3 and RPi0W so far, i would suggest to check it against `is_pithreeplus`

Reply:
In my effort to merge  the build scripts and having a conditional first-boot.sh for all platforms I have used the platform identifications functions as `is_pithreeplus`. I will make a note on condition this code in that pull request. https://github.com/openhab/openhabian/pull/358